ubiquibot-config-default.yml

```
@@ -0,0 +1,41 @@
+---
+chain-id: 1
```

In this context I fear that network ID might not be as clear as `chain-id` but I want to conform it across the entire infrastructures' codebase because I still think its semantically incorrect to refer to rollups as "chains"

```suggestion
evm-network-id: 1
```